### PR TITLE
Simplify and yet improve the description

### PR DESCRIPTION
**Pull request by vmoutoussamy**
And also mention SLES.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # jeos-firstboot
 ## Description
-jeos-firstboot is a lightweight, dialog based firstboot wizard that replaces systemd's line based firstboot program. It can show the license and prompt for language, keyboard, timezone, root passsword and network configuration.
+jeos-firstboot allows initial configuration and adjustments of a Linux system using text based dialogs.
 
-This was mainly developed to work with openSUSE JeOS images, more information visit [JeOS wiki](https://en.opensuse.org/Portal:JeOS)
+It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. Including showing the license and prompt for language, keyboard, timezone, root passsword and network configuration..
+
+This was mainly developed to work with openSUSE JeOS and SUSE Linux Enterprise Server JeOS images, more information visit [JeOS wiki](https://en.opensuse.org/Portal:JeOS)
```
Comment (Vogtinator) [inline README.md:7]:
```suggestion
This was mainly developed for openSUSE and SUSE Linux Enterprise Server JeOS images. For more information visit the [JeOS wiki](https://en.opensuse.org/Portal:JeOS).
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
 # jeos-firstboot
 ## Description
-jeos-firstboot is a lightweight, dialog based firstboot wizard that replaces systemd's line based firstboot program. It can show the license and prompt for language, keyboard, timezone, root passsword and network configuration.
+jeos-firstboot allows initial configuration and adjustments of a Linux system using text based dialogs.
 
-This was mainly developed to work with openSUSE JeOS images, more information visit [JeOS wiki](https://en.opensuse.org/Portal:JeOS)
+It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. Including showing the license and prompt for language, keyboard, timezone, root passsword and network configuration..
```
Comment (Vogtinator) [inline README.md:5]:
```suggestion
It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. This includes showing the license and prompt for language, keyboard layout, timezone, root passsword and configuring the network.
```

Comment (gmoro) [inline README.md:5]:
```suggestion
It is a lightweight and customisable firstboot wizard that allows to set basic system settings during and after the first boot of an image. Including showing the license and prompt for language, keyboard, timezone, root passsword and network configuration.
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
+
+This was mainly developed for openSUSE and SUSE Linux Enterprise Server JeOS images. For more information visit the [JeOS wiki](https://en.opensuse.org/Portal:JeOS).
```
Comment (gmoro) [inline README.md:7]:
```suggestion
This is mainly developed for openSUSE and SUSE Linux Enterprise Server JeOS images. For more information visit the [JeOS wiki](https://en.opensuse.org/Portal:JeOS).
```

**Review by gmoro — COMMENTED** [review]
Looks good, @Vogtinator and @vmoutoussamy may I merge it?

**Pull request merged**